soc/ti/hercules/ti_tms570lc43x/soc.c

```diff
@@ -0,0 +1,334 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
```

Comment:
We typically cannot have BSD-3-Clause licensed code in the tree, as this needs to be approved by the governing board (see https://docs.zephyrproject.org/latest/contribute/guidelines.html#components-using-other-licenses).

Reply:
moved the BSD licensed code to hal_ti module (PR here https://github.com/zephyrproject-rtos/hal_ti/pull/63)